### ISSUE TITLE
v2和fluid 1.0训练速度差别问题

### ISSUE THREAD

**Issue by shiyazhou121** [PaddlePaddle/Paddle#14135]
我在集群上分别用V2和fluid 1.0实现训练，V2版本的跑10个epoch需要10个小时左右，但fluid版本的已经跑了24个小时了，才跑到第7个epoch，想问下是我代码写的逻辑问题还是其他的，应该怎么优化？
以下是fluid版的部分代码：
 emb = fluid.layers.embedding(
        input=feature, 
        size=[input_dim, emb_dim], 
        is_sparse=True)
    conv_1 = fluid.nets.sequence_conv_pool(
        input=emb,
        num_filters=hid_dim,
        filter_size=1,
        act="tanh",
        pool_type="average")
    conv_2 = fluid.nets.sequence_conv_pool(
        input=emb,
        num_filters=hid_dim,
        filter_size=2,
        act="tanh",
        pool_type="average")
    conv_3 = fluid.nets.sequence_conv_pool(
        input=emb,
        num_filters=hid_dim,
        filter_size=3,
        act="tanh",
        pool_type="average")
    pred = fluid.layers.fc(input=[conv_1, conv_2, conv_3],
                                 size=class_dim,
                                 act="softmax")
    cost = fluid.layers.cross_entropy(input=pred, label=label)
    avg_cost = fluid.layers.mean(cost)
    acc = fluid.layers.accuracy(input=pred, label=label)

test_program = fluid.default_main_program().clone(for_test=True)
    adam_optimizer = fluid.optimizer.Adam(
        learning_rate=4e-3)
    adam_optimizer.minimize(avg_cost)

for pass_id in range(EPOCHS):
            # train in the epoch
            for batch_id, data in enumerate(train_dataReader()):
                #comp.reset()
                r_cost, r_acc, r_pred, r_label = exe.run(
                    program=main_program, 
                    feed=feeder.feed(data), 
                    fetch_list = [avg_cost, acc, pred, label])

**Comment by jacquesqiao** [comment]
这个exe是单线程的，改成parallel_exe吧

**Comment by dzhwinter** [comment]
参考一下models里的使用方式，用parallel_executor吧

**Comment by shiyazhou121** [comment]
@dzhwinter 是在mpi-paddle集群上跑的程序，也需要改成parallel吗？

**Comment by dzhwinter** [comment]
parallel executor是多线程，多卡的，executor是单线程的。看你的应用场景

**Comment by lucywsq** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持!
